client/src/app/site/pages/meetings/pages/motions/components/motion-forward-dialog/components/motion-forward-dialog/motion-forward-dialog.component.html

```diff
@@ -18,6 +18,10 @@ <h4>{{ committee.name }}</h4>
                         }}"
                     >
                         {{ meeting.name }}
+                        <span *ngIf="meeting.start_time && meeting.end_time">
```

Comment:
I think it is possible to set just the start or the end time, right? So this should be split to only display one of them